src/core.c/Int.pm6

```diff
@@ -26,6 +26,10 @@ my class Int does Real { # declared in BOOTSTRAP
         )
     }
 
+    multi method ACCEPTS(Int:D: Int:D \other, --> Bool:D) {
+        nqp::hllbool(nqp::iseq_I(self, nqp::decont(other)))
```

Comment:
I think it is nowadays better to just do `Int:D $other` and lose the explicit decont.